src/sage/combinat/free_module.py

```diff
@@ -1361,7 +1361,7 @@ def __init__(self, modules, **options):
             """
             self._sets = modules
             indices = CartesianProduct_iters(*[module.basis().keys()
-                                               for module in modules]).map(tuple)
+                                               for module in modules]).map(tuple, is_injective=True)
```

Comment:
why this change?  `is_injective` is the default, and also what I'd assume that `map` does.

Reply:
just for clarity

Reply:
> also what I'd assume that `map` does.

That's one of the trickier parts in the enumerated sets code. See e.g. #34389